interfaces/HO-Portal/src/app/pages/payment/payment.page.html

```diff
@@ -12,12 +12,20 @@
     [phaseReady]="isReady"
   ></app-phase-next>
 
+  <ion-row *ngIf="!isIntersolve">
```

Comment:
You'd want to show this ONLY if the program uses Intersolve as an FSP, right? (So without the `!`)?